### DISCUSSION TITLE
blog/document/js/jsShow

### DISCUSSION THREAD

**Discussion by giscus** [dyywork/blog#13] [Announcements]
# blog/document/js/jsShow

js 原生小效果

http://localhost:8888/blog/document/js/jsShow.html

**Comment by dyywork** [comment]
真好玩！！！

**Comment by dyywork** [comment]
梵蒂冈黑胡椒和餐具即可v哦个65555643

**Comment by dyywork** [comment]
13五(⊙﹏⊙)如同太阳和依靠里75645

**Comment by dyywork** [comment]
五千万人提供话估计快了【【【【【【】

**Comment by dyywork** [comment]
改变面貌，，；。nmvgzwq33e45656yghghkjkjmjkmmn
